app/build.gradle

```diff
@@ -58,4 +58,9 @@ dependencies {
     implementation platform('com.google.firebase:firebase-bom:28.2.1')
     implementation 'com.google.firebase:firebase-analytics-ktx'
     implementation 'com.google.firebase:firebase-firestore-ktx'
+
+    def room_version = "2.3.0"
```

Comment:
Missing imports in the gradle for RoomDB. Documentation and other optional add-ons available here: https://developer.android.com/training/data-storage/room/